lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -549,7 +552,53 @@ def upload_csv_to_report_store(rows, csv_name, course_id, timestamp):
     )
 
 
-def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input, action_name):
+def verification_status_for_user(user, course_id, user_enrollment_mode):
```

Comment:
I am thinking that it would make more sense to make this a class method of `SoftwareSecurePhotoVerification` and make the docstring: "Return the verification status for use in grade reports."